WebRoot/public/admin.jsp

```diff
@@ -196,7 +196,8 @@
 <script type="text/javascript" src="${contextPath}/yui/2.7.0/element/element-min.js"></script> 
 <script type="text/javascript" src="${contextPath}/yui/2.7.0/datasource/datasource-min.js"></script> 
 <script type="text/javascript" src="${contextPath}/yui/2.7.0/json/json-min.js"></script> 
-<script type="text/javascript" src="${contextPath}/yui/2.7.0/charts/charts-min.js"></script> 
+<script type="text/javascript" src="${contextPath}/yui/2.7.0/charts/charts-min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js"></script>
```

Comment:
@silentsakky I need your help here. What is the proper way to include third-party libs in admin-console environment?

Reply:
Fixed.